GunShipAlpha/src/Enemy.java

```diff
@@ -162,3 +162,33 @@ private ArrayList<ExplosionParticle> createExplosionParticles() {
         return particles;
     }
 }
+
+class Ginker extends Enemy{
+
+    public Ginker(int x, int y, int targetX, int targetY) {
+        super(x, y, targetX, targetY);
+        this.sprites = new GinkerSprite();
```

Comment:
see above related to passing the name of the sprite path to the superclass: 

super(x, y, targetX, targetY, "Ginker");

also, I thought there was earlier feedback about making x/y a point, and making targetX/targetY a point? I'd like to see less parameters passed around everywhere when they clearly are part of the same "structure".